crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/controller/MeansAssessmentIntegrationTest.java

```diff
@@ -55,7 +55,8 @@
 class MeansAssessmentIntegrationTest {
 
     private static final boolean IS_VALID = true;
-    private static final String ENDPOINT_URL = "/api/internal/v1/assessment/means";
+    private static final String API_VERSION = "/api/internal/v1/assessment";
+    private static final String ENDPOINT_URL = API_VERSION + "/means";
```

Comment:
How about just having one constant for the authorization endpoints and another for the means?